bedrock/firefox/templates/firefox/whatsnew/fx64/whatsnew-fx64-base.html

```diff
@@ -88,11 +88,11 @@ <h3>{{ notes_head }}</h3>
       <p><a class="mzp-c-button wn64-benefit-link" data-app="notes" href="https://play.google.com/store/apps/details?id=org.mozilla.testpilot.notes">{{ notes_link }}</a></p>
     </div>
 
-    <div class="wn64-benefit wn64-lockbox">
-      {{ high_res_img('firefox/whatsnew_64/fxa-lockbox.png', {'alt': '', 'width': '590', 'height': '590'}) }}
-      <h3>{{ lockbox_head }}</h3>
-      <p>{{ lockbox_desc }}</p>
-      <p><a class="mzp-c-button wn64-benefit-link" data-app="lockbox" href="https://app.adjust.com/orzhlh0">{{ lockbox_link }}</a></p>
+    <div class="wn64-benefit wn64-lockwise">
+      {{ high_res_img('firefox/whatsnew_64/fxa-lockwise.png', {'alt': '', 'width': '590', 'height': '590'}) }}
```

Comment:
The low res version of this file was renamed but the high-res on needs the rename too.